apps/sample-app/src/PageLayout.jsx

```diff
@@ -0,0 +1,19 @@
+/* eslint-disable @typescript-eslint/explicit-function-return-type */
+import { useIsAuthenticated } from '@azure/msal-react';
+import React from 'react';
+
+import { SignInButton } from './components/Button';
+
+// eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
+export const PageLayout = props => {
+  const isAuthenticated = useIsAuthenticated();
+
+  return (
+    <>
+      {isAuthenticated ? <span>Signed In</span> : <SignInButton />}
```

Comment:
you're using `SignInButton` as if it was a react component, but I don't think it is.